homeassistant/components/waze_travel_time/sensor.py

```diff
@@ -204,6 +206,7 @@ def update(self):
                     }
 
                 if routes:
+                    _LOGGER.error(routes)
```

Comment:
Same as above, I think it should be a debug level

Reply:
Thanks for catching that!